src/SideNav.js

```diff
@@ -56,6 +72,8 @@ SideNav.propTypes = {
     closeOnClick: PropTypes.bool,
     draggable: PropTypes.bool
   }),
+  fixed: PropTypes.bool,
+  showOnLarge: PropTypes.bool,
```

Comment:
I know this file doesn't have many comments on prop-types but could we had here a little comment saying what this prop does? like [here](https://github.com/react-materialize/react-materialize/blob/9470eafc655a4b3bb41a0971fe23ad8a6e7f71ff/src/NavItem.js#L22)